backend/consensus/vrf.py

```
@@ -1,16 +1,20 @@
 from datetime import datetime
 import numpy as np
+from backend.consensus.base import DEFAULT_VALIDATORS_COUNT
 
 
 def get_validators_for_transaction(
     nodes: list[dict],
-    num_validators: int,
+    num_validators: int | None = None,
     rng=np.random.default_rng(seed=int(datetime.now().timestamp())),
```

Ruff (0.11.9)

Do not perform function call `np.random.default_rng` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)